src/plugins/lvm-dbus.c

```diff
@@ -2239,7 +2255,7 @@ BDLVMLVdata** bd_lvm_lvs (const gchar *vg_name, GError **error) {
         return NULL;
     }
     success = filter_lvs_by_vg (lvs, vg_name, &matched_lvs, &n_lvs, error);
-    g_free (lvs);
+    g_strfreev ((gchar **) lvs);
```

Comment:
The typecast shouldn't be here, the `get_existing_objects()` function should not return `const gchar**` in the first place since it returns new fully allocated array, no sign of reusing foreign memory that would qualify it for `const`.

Could you fix it altogether please?